rllib/env/wrappers/atari_wrappers.py

```diff
@@ -221,9 +219,8 @@ def __init__(self, env, dim):
             dtype=np.uint8)
 
     def observation(self, frame):
-        frame = cv2.cvtColor(frame, cv2.COLOR_RGB2GRAY)
-        frame = cv2.resize(
-            frame, (self.width, self.height), interpolation=cv2.INTER_AREA)
+        frame = color.rgb2gray(frame)
+        frame = transform.resize(frame, (self.height, self.width))
```

Comment:
Can we make cv2 optional?

For example, if you detect CV2, then you can import it. Otherwise, use skimage.

Reply:
The benefit is that this avoids any backwards incompatible changes.

Reply:
I can do that, but what would be backwards incompatible? 
With the change the code should work exactly the same and CV2 is not a dependency currently, so it has to be installed manually anyway.
So if skimage is added as an RLlib dependency then there should be no issue I would assume, but I might be missing something.

Reply:
Hmm, I suspect that the interpolation mechanism used by skimage is not the same as cv2. This may result in user models performing differently. If possible, it'd be nice to make sure the cv2 codepath is not removed, but its ok to make it a secondary option.

Reply:
Ok, I'll get that sorted then. I'll add some generic functions like
```python
def scale(img, ...):
    if cv2:
        scale with cv2
    elif skimage:
        scale with skimage
    else:
        error

def rgb2gray(...):
    ...
```
Otherwise the code gets a bit cluttered I think.
Does that make sense?

Reply:
+1!